file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
Might be good to ask @lutter / @scottyw / @LarissaLane here, because I'm just coming up to speed on how the translation workflow actually works.